### PR TITLE
feat: Implement breadcrumb, back and forward buttons

### PR DESCRIPTION
**Pull request by olatundeee**
Added a new component `TopNavbar` that handles the dynamic breadcrumb functionality

### CODE REVIEW

File: src/renderer/components/Navbar.js [hunk 1]
```diff
@@ -17,128 +17,132 @@ class SideBar extends Component {
...
-                    <Nav.Link href="#/trends">
-                        <div className="nav_icons"><img src={iconTrend} height="21px" />
-                        </div>
-                            Trending Content
-                    </Nav.Link>
-                    <Nav.Link href="#/new">
-                        <div className="nav_icons"><img src={iconNewContent} height="17px" /></div>
-                        New Content
+        return (
+            <div>
```
Comment (vaultec81) [inline src/renderer/components/Navbar.js]: Any particular reason why you added an extra div tag enclosing the navbar?

File: src/renderer/components/TopNavbar.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    render() {
+        return (
+           <div>
+                 <Navbar bg="light" expand="lg">
+                    <Navbar.Collapse id="basic-navbar-nav">
+                    <Nav className="mr-auto">
+                    <Breadcrumb>
+                        <Breadcrumb.Item href="/">Home</Breadcrumb.Item>
```
Comment (vaultec81) [inline src/renderer/components/TopNavbar.js]: We use react router's hashrouter, which means all links must start with `#/` including the root directory.

File: src/renderer/components/TopNavbar.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                        {this.state.urlSplit.map(el => (
+                            <Breadcrumb.Item href="#" key={el}>{el}</Breadcrumb.Item>
```
Comment (vaultec81) [inline src/renderer/components/TopNavbar.js]: Make sure to dynamically generate the href from url.

File: src/renderer/components/TopNavbar.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        super(props);
+        this.state = {urlSplit: []};
+    }
+    
+    HeaderView() {
+        let urlSplit;
+        const windowLocation = window.location.href
+        
+        urlSplit = windowLocation.split('/')
+        const newUrl = urlSplit.splice(0, 4);
```
Comment (vaultec81) [inline src/renderer/components/TopNavbar.js]: Unused variable?

**Review by vaultec81 — CHANGES_REQUESTED** [review]
Checklist
- [x] Resolve merge conflicts
- [x] Implement forward and back buttons
- [x] Implement an ability to copy current URL and paste URL into navbar (In doing so would direct to the page of input)
- [x] Revert any Linters/prettifiers
- [x] Provide screenshot of changes

A small note on linting. Generally speaking I've seen the use of excessive prettifiers/linters problematic in large projects. Please keep the usage of them at a minimum. I will be adding a universal lint command sometime soon.

File: src/renderer/components/Navbar.js (hunk 1 above)
Comment (olatundeee) [reply src/renderer/components/Navbar.js]: I have fixed this

File: src/renderer/components/TopNavbar.js (hunk 3 above)
Comment (olatundeee) [reply src/renderer/components/TopNavbar.js]: Done

File: src/renderer/components/TopNavbar.js (hunk 4 above)
Comment (olatundeee) [reply src/renderer/components/TopNavbar.js]: Done

File: src/renderer/App.js [hunk 5]
```diff
@@ -11,33 +11,29 @@ import views from './views';
...
+import TopNavbar from './components/TopNavbar';
 
 export class App extends React.PureComponent {
   render() {
     return (
       <div>
-        <Sidebar />
-        <HashRouter>
-          <Switch>
-            <Route path="/" exact>
```
Comment (vaultec81) [inline src/renderer/App.js:22]: There appears to be a regression here removing routes from main branch

File: src/renderer/components/TopNavbar.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+    render() {
+        let windowLocationUrl = window.location.href
+        let windowLocationSearch = windowLocationUrl.search('#')
+        let windowLocationHref = windowLocationUrl.slice(windowLocationSearch)
+
+        function copyToClip() {
+            navigator.clipboard.writeText(windowLocationUrl)
+            .then(() => {
+                console.log('done',);
```
Comment (vaultec81) [inline src/renderer/components/TopNavbar.js]: Unnecessary `,`

**Review by vaultec81 — CHANGES_REQUESTED** [review]

**Review by vaultec81 — APPROVED** [review]

**Pull request merged**